README.md

```diff
@@ -20,7 +20,7 @@
 [license-url]: https://github.com/tldr-pages/tldr/blob/master/LICENSE.md
 [license-image]: https://img.shields.io/github/license/tldr-pages/tldr.svg
 
-A collection of simplified and community-driven man pages.
```

Comment:
I think I like the "A collection of" better in the README, but adding the "and community-driven" to the tldr page is a great addition

Reply:
There's a lot of room for discussion of that in #1192 (I supported some more creative ideas there), but I'd keep the description consistent everywhere. That also makes future changes easier, as we can just search for this description and replace it.

Reply:
@zdroid the description in the README and the one of the page of the command do *not* need to be the same. The `tldr.md` page is meant to relate to the command line `tldr` client, not the whole project.

Reply:
Our clients have the same description, though.

Reply:
If anything, I would change the description in the page to:

    > Command-line client for tldr pages: simplified and community-driven man pages.

Having only the second part doesn't even describe what the program is for.

Reply:
Good idea @mebeim. Others?

Reply:
That sounds good, @mebeim!

Reply:
Done @mebeim.

Reply:
Can we keep the `A collection of simplified` here please?

Reply:
I find this somewhat long and repetitive. Because we have "... tldr pages: .. man pages". Can we have a single sentence which flows better ?

I feel that "Command-line client for tldr pages" should be good enough. :man_shrugging:

Reply:
This isn't the CLI description though, @agnivade? I'm confused.

Reply:
Ok, how about we put the description on a new line. And keep the first line as "Command-line client for tldr pages".